movies-app/src/components/features/shows/ShowContainer/ShowContainer.tsx

```diff
@@ -51,21 +51,22 @@ const mockReviewItems: IReviewList = {
 export default function ShowContainer() {
   const toast = useToast();
   const params = useParams();
-  const [reviewsList, setReviewsList] = useState(mockReviewItems);
-  const { data, error, isLoading } = useSWR(`/api/shows/${params.id}`, ()=> getShow(params.id as string));
 
-  const { data: reviewsData, error: reviewsError, isLoading: reviewsIsLoading } = useSWR(swrKeys.showReviews(params.id as string), ()=> getShowReviews(params.id as string));
+  const { data, error, isLoading } = useSWR(`/api/shows/${params.id}`, () =>
+    getShow(params.id as string)
+  );
 
-  console.log(reviewsData);
+  const {
+    data: reviewsData,
+    error: reviewsError,
+    isLoading: reviewsIsLoading,
+  } = useSWR(swrKeys.showReviews(params.id as string), () =>

```

Comment:
I feel like this could easily be in the ShowReviewSection or a similar component.